Doc/whatsnew/3.12.rst

```diff
@@ -1020,9 +1027,27 @@ CPython bytecode changes
 * Removed the :opcode:`!PRECALL` instruction. (Contributed by Mark Shannon in
   :gh:`92925`.)
 
+* Add the :opcode:`BINARY_SLICE` and :opcode:`STORE_SLICE` instructions.
+  (Contributed by Mark Shannon in :gh:`94163`.)
+
+* Add the :opcode:`!CALL_INSTRINSIC_1` instructions.
+  (Contributed by Mark Shannon in :gh:`99005`.)
+
+* Add the :opcode:`!CALL_INSTRINSIC_2` instruction.
+  (Contributed by Irit Katriel in :gh:`101799`.)
+
+* Add the :opcode:`CLEANUP_THROW` instruction.
+  (Contributed by Brandt Bucher in :gh:`90997`.)
+
+* Add the :opcode:`!END_SEND` instruction.
+  (Contributed by Mark Shannon in :gh:`103082`.)
```

Comment:
Can we document `CALL_INSTRINSIC_1`, `CALL_INSTRINSIC_2` and `END_SEND` opcodes on the `dis` module page, like the others, and then link to them?

Reply:
This is @MatthieuDartiailh's PR (#108900). Though CALL_INTRINSIC_1/2 seem to already be documented, unsure why these weren't being picked up.

A